metastore/src/test/scala/com/gu/tableversions/metastore/VersionPathsSpec.scala

```diff
@@ -0,0 +1,55 @@
+package com.gu.tableversions.metastore
+
+import java.net.URI
+
+import com.gu.tableversions.core.Partition.PartitionColumn
+import com.gu.tableversions.core.{Partition, PartitionVersion, VersionNumber}
+import org.scalatest.{FlatSpec, Matchers}
+
+class VersionPathsSpec extends FlatSpec with Matchers {
+
+  "Resolving a path with version 0" should "produce the original path" in {
```

Comment:
Is this capturing the use case for migrating i.e. not moving all of our data into `v0` prefixes?

Reply:
That's the intention, yes.